experimental/DoubleAndHyperComplexes/src/Morphisms/simplified_complexes.jl

```diff
@@ -496,7 +496,7 @@ function _simplify_matrix!(A::SMat; find_pivot=nothing)
     # clear the q-th column
     for (i, b) in a_col_del
       #A[i] = A[i] - uinv*b*a_row # original operation, replaced by inplace arithmetic below
-      Hecke.add_scaled_row!(a_row, A[i], -uinv*b)
+      addmul!(A[i], a_row, -b*uinv)
```

Comment:
Why was `-uinv*b` changed?

Reply:
Typo

Reply:
ok, I thought it was some left/right thing